client-rpc/src/wallet_rpc.rs

```diff
@@ -20,7 +20,7 @@ pub trait WalletRpc {
     fn addresses(&self, request: WalletRequest) -> jsonrpc_core::Result<Vec<String>>;
 
     #[rpc(name = "wallet_balance")]
-    fn balance(&self, request: WalletRequest) -> jsonrpc_core::Result<Coin>;
+    fn balance(&self, request: WalletRequest) -> jsonrpc_core::Result<String>;
```

Comment:
any reason why not change the serde implementations as in @calvinaco 's old PR https://github.com/crypto-com/chain/pull/122/files#diff-8fb93dcdc2ace21f62aa8ef871ba2102 ?

Reply:
i'll start from calvain' pr again.